lib/public/components/common/selection/picker/picker.js

```diff
@@ -94,16 +98,23 @@ export const picker = (pickerOptions, pickerModel, configuration) => {
         }
     };
 
-    return [
-        selectedPills,
-        h(`input.form-control.${selector}-search-input`, {
-            type: 'search',
-            placeHolder: '🔍 Search for tags',
-            value: pickerModel.searchInputContent,
-            oninput: (e) => {
-                pickerModel.searchInputContent = e.target.value;
-            },
-        }),
-        applyAttributes(checkboxes.length ? checkboxes : h('em', 'No options')),
-    ];
+    if (searchEnabled) {
```

Comment:
Is it not better to simply add the `searchEnabled` check in front of the `h` element of the search?
Or event better extract the `h(input [...])` into its own component and then use:
```js
searchEnabled && searchInputElement()
```